### ISSUE TITLE
hidesBarsOnTap does not work

### ISSUE THREAD

**Issue by litt1e-p** [HackingGate/PDF-Reader#2]
perhaps use hidesBarsOnSwipe?

**Comment by HackingGate** [comment]
`pdfView.scrollView?.gestureRecognizers` are causing `hidesBarsOnTap` not to work on iOS 13 and above.

Change `navigationController?.barHideOnTapGestureRecognizer.numberOfTouchesRequired` to 2 is a prefect fix in this situation.

**Comment by HackingGate** [comment]
Apple PDFKit is embedded with iOS Systems. Not with project.
In my project. `hidesBarsOnTap` works on iOS 11 and iOS 12. 

I tried the answer here:
https://stackoverflow.com/a/54986986/4063462
```
func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
    return true
}
```
It did not work.

I also tried to remove all `gestureRecognizers` excluding UIScrollView built-in gestures from  `pdfView.scrollView?`. `hidesBarsOnTap` somehow works but unstable. 
Altering those `gestureRecognizers` could cause many other problems. So I stopped there.
